<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormtwoLegStrategy</class>
 <widget class="QWidget" name="FormtwoLegStrategy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Two Leg Startegy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox1LSide">
     <item>
      <property name="text">
       <string>Buy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sell</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxMinProfit">
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>1st Leg Side</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox1LSymbol"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox2LSide">
     <item>
      <property name="text">
       <string>Buy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sell</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBoxLots">
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>2nd Leg Instrument</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Min Profit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>1st Leg Instrument</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox2LSymbol"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Lots</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>2ndLeg Side</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Reload Symbols</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
